server/routes/search/handlers/probationSearch.ts

```diff
@@ -1,42 +1,54 @@
 import { Request, Response } from 'express'
 import SearchService from '../../../services/searchService'
 import statusConfig from '../../../licences/licenceStatus'
+import type { ProbationSearchResult } from '../../../@types/licenceApiClientTypes'
 
 export default class ProbationSearch {
   constructor(private readonly searchService: SearchService) {}
 
   GET = async (req: Request, res: Response): Promise<void> => {
     const queryTerm = req.query.queryTerm as string
     const { deliusStaffIdentifier } = res.locals.user
-    const previousPage = req.query.previousPage as string
+    const previousCaseloadPage = req.query.previousPage as string
 
-    const searchResponse = await this.searchService.getProbationSearchResults(queryTerm, deliusStaffIdentifier)
-    const searchResults = searchResponse.results
-    const backLink = this.getBackLink(previousPage)
+    let searchResponse: ProbationSearchResult
 
-    let inPrisonCountText: string
-    let onProbationCountText: string
-
-    if (searchResponse.inPrisonCount === 0 || searchResponse.inPrisonCount > 1) {
-      inPrisonCountText = `People in prison (${searchResponse.inPrisonCount} results)`
+    if (queryTerm.length === 0) {
+      searchResponse = {
+        results: [],
+        inPrisonCount: 0,
+        onProbationCount: 0,
+      }
     } else {
-      inPrisonCountText = `People in prison (${searchResponse.inPrisonCount} result)`
+      searchResponse = await this.searchService.getProbationSearchResults(queryTerm, deliusStaffIdentifier)
     }
 
-    if (searchResponse.onProbationCount === 0 || searchResponse.onProbationCount > 1) {
-      onProbationCountText = `People on probation (${searchResponse.onProbationCount} results)`
+    const backLink = this.getBackLink(previousCaseloadPage)
+
+    let activeTab: string
+
+    if (searchResponse.onProbationCount === 0 || searchResponse.inPrisonCount >= searchResponse.onProbationCount) {
```

Comment:
is the `searchResponse.onProbationCount === 0 ` clause necessary? 

There's generally quite a lot of mutable references in the PR, doesn't require action now but it's generally worth trying to avoid them wherever possible.. there's a few ways around this - ternarys, "or" ing on falsy values, or extracting functions, or removing the conditional entirely